plugins/ccgx/fu-ccgx-dmc-device.c

```diff
@@ -318,6 +319,7 @@ fu_ccgx_dmc_device_to_string(FuDevice *device, guint idt, GString *str)
 				   fu_ccgx_fw_image_type_to_string(self->fw_image_type));
 	fu_common_string_append_kx(str, idt, "EpBulkOut", self->ep_bulk_out);
 	fu_common_string_append_kx(str, idt, "EpIntrIn", self->ep_intr_in);
+	fu_common_string_append_kx(str, idt, "Trigger Code", self->trig_code);
```

Comment:
`Trigger`Code` here please

Reply:
I will change it to TriggerCode

Reply:
Please check [53c653c](https://github.com/fwupd/fwupd/pull/4435/commits/53c653ca9db90a142b95992e167f06f819396a0f)